### ISSUE TITLE
Relic KL_div

### ISSUE THREAD

**Issue by shixishi** [NightShade99/Self-Supervised-Vision#5]
in class RelicLoss 
        probs_io = F.softmax(logits_io[torch.logical_not(mask)], -1)
        probs_jo = F.log_softmax(logits_jo[torch.logical_not(mask)], -1)
        kl_div_loss = F.kl_div(probs_io, probs_jo, log_target=True, reduction="sum")
 probs_io should be applied F.log_softmax, not F.softmax

**Comment by NightShade99** [comment]
Hello! Really sorry for the extremely delayed response 😓. I had kinda forgotten about this repo a couple years ago, but came back to it today looking for a code snippet, and noticed your issue.

Coming to your issue: yes, that correction is valid, it should be `F.log_softmax`.
